packages/desktop-client/src/components/modals/AccountMenuModal.tsx

```diff
@@ -1,8 +1,11 @@
-import React, { type ComponentProps, useRef, useState } from 'react';
+import React, { type ComponentProps, Fragment, useRef, useState } from 'react';
+
+import { t } from 'i18next';
 
```

Comment:
Same comment here

Reply:
Okay. I just was applying it, assuming it was the same given it's use in the codebase, though now after reading, I think I understand the differences more.

Reply:
Both are updated.